test_communication/package.xml

```diff
@@ -11,21 +11,23 @@
   <maintainer email="dthomas@osrfoundation.org">Dirk Thomas</maintainer>
   <license>Apache License 2.0</license>
 
+  <build_depend>ament_cmake_auto</build_depend>
+  <build_depend>builtin_interfaces</build_depend>
+  <build_depend>rclcpp</build_depend>
+  <build_depend>rclpy</build_depend>
+  <build_depend>rcl</build_depend>
```

Comment:
(forgot to actually submit this comment with the review, sorry)

Do these (`rcl*`) packages need to be build depends? I think they could stay as test depends.